docs/about/release-notes.rst

```diff
@@ -36,8 +36,8 @@ Release Notes
    and Sunyoung Yoo :sup:`a`
 
 
-v23.xy.0 (January 2023)
------------------------
+Version NEXT_RELEASE
+---------------
```

Comment:
The line will be too short if we ever have more than 10 releases in one month. Seems unlikely but you never know what happens during hot commissioning.

Reply:
Yes. Solving this would mean that the tox script would have to insert the dashes. But we can cross that bridge when we get there.